scripts/onboarding/aks/multi-tenancy/existingClusterOnboarding.json

```diff
@@ -0,0 +1,178 @@
+            }
+        },
+        "k8sNamespaces": {
+            "type": "array",
+            "metadata": {
+                "description": "An array of Kubernetes namespaces for Multi-tenancy logs filtering"
+            }
+        },
+        "transformKql": {
+            "type": "string",
+            "metadata": {
+                "description": "KQL filter for ingestion transformation"
+            }
+        }
+    },
+    "variables": {
+        "clusterSubscriptionId": "[split(parameters('aksResourceId'),'/')[2]]",
+        "clusterResourceGroup": "[split(parameters('aksResourceId'),'/')[4]]",
+        "clusterName": "[split(parameters('aksResourceId'),'/')[8]]",
+        "clusterLocation": "[replace(parameters('aksResourceLocation'),' ', '')]",
+        "workspaceName": "[split(parameters('workspaceResourceId'),'/')[8]]",
+        "workspaceLocation": "[replace(parameters('workspaceRegion'),' ', '')]",
+        "dcrNameFull": "[Concat('MSCI-multi-tenancy', '-', variables('workspaceLocation'), '-', uniqueString(parameters('workspaceResourceId')))]",
+        "dcrName": "[if(greater(length(variables('dcrNameFull')), 64), substring(variables('dcrNameFull'), 0, 64), variables('dcrNameFull'))]",
+        "associationName": "[Concat('ContainerLogV2Extension', '-', uniqueString(parameters('workspaceResourceId')))]",
+        "dataCollectionRuleId": "[resourceId(variables('clusterSubscriptionId'), variables('clusterResourceGroup'), 'Microsoft.Insights/dataCollectionRules', variables('dcrName'))]",
+        "ingestionDCENameFull": "[Concat('MSCI-multi-tenancy', '-', variables('workspaceLocation'), '-', uniqueString(parameters('workspaceResourceId')))]",
+        "ingestionDCEName": "[if(greater(length(variables('ingestionDCENameFull')), 43), substring(variables('ingestionDCENameFull'), 0, 43), variables('ingestionDCENameFull'))]",
+        "ingestionDCE": "[if(endsWith(variables('ingestionDCEName'), '-'), substring(variables('ingestionDCEName'), 0, 42), variables('ingestionDCEName'))]",
+        "ingestionDataCollectionEndpointId": "[resourceId(variables('clusterSubscriptionId'), variables('clusterResourceGroup'), 'Microsoft.Insights/dataCollectionEndpoints', variables('ingestionDCE'))]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Insights/dataCollectionEndpoints",
+            "apiVersion": "2022-06-01",
+            "name": "[variables('ingestionDCE')]",
+            "location": "[variables('workspaceLocation')]",
+            "tags": "[parameters('resourceTagValues')]",
+            "kind": "Linux",
+            "properties": {
+                "networkAcls": {
+                    "publicNetworkAccess": "Enabled"
+                }
+            }
+        },
+        {
+            "type": "Microsoft.Resources/deployments",
+            "name": "[Concat('aks-monitoring-msi-dcr', '-',  uniqueString(variables('dcrName')))]",
+            "apiVersion": "2017-05-10",
```

Comment:
do we want to use new api version?

Reply:
> do we want to use new api version?

update the api version for DCR, DCE and DCR-A to latest version.